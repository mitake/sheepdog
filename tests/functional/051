#!/bin/bash

# Test cluster with too few living nodes

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_cleanup

for i in 0 1; do
    _start_sheep $i
done
_wait_for_sheep 2
_cluster_format
$COLLIE cluster info | _filter_cluster_info

_start_sheep 2
_wait_for_sheep 3
$COLLIE cluster info | _filter_cluster_info

_kill_sheep 2
_wait_for_sheep 2
$COLLIE cluster info | _filter_cluster_info
