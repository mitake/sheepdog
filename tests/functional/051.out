QA output created by 051
using backend plain store
Cluster status: running, auto-recovery enabled

Cluster created at DATE

Epoch Time           Version
DATE      1 [127.0.0.1:7000, 127.0.0.1:7001]
Cluster status: running, auto-recovery enabled

Cluster created at DATE

Epoch Time           Version
DATE      2 [127.0.0.1:7000, 127.0.0.1:7001, 127.0.0.1:7002]
DATE      1 [127.0.0.1:7000, 127.0.0.1:7001]
Cluster status: running, auto-recovery enabled

Cluster created at DATE

Epoch Time           Version
DATE      3 [127.0.0.1:7000, 127.0.0.1:7001]
DATE      2 [127.0.0.1:7000, 127.0.0.1:7001, 127.0.0.1:7002]
DATE      1 [127.0.0.1:7000, 127.0.0.1:7001]
